vmb/matrimony/models/profiles.py

```diff
@@ -400,6 +402,15 @@ class MatrimonyProfile(BaseModel):
     )
     comments = GenericRelation("Comment")
 
+    @property
+    def annual_income_in_inr(self):
+        currency_exchange, created = CurrencyExchangeRate.objects.get_or_create(
+            from_currency=str(self.annual_income.currency)
+        )
```

Comment:
When creating a new CurrencyExchange object, shouldn't you be fetching rates for that currency pair?

Reply:
yes, that is been taken care by the get_exchange_rate function in CurrencyExchangeRate

Reply:
the line after that is calling get_exchange_rate() which is taking care of fetching rates